Earthfile

```diff
@@ -1,5 +1,5 @@
 VERSION 0.8
-PROJECT earthly-technologies/core
```

Comment:
I assume you had to change this to get it working locally? We should probably revert it back

Reply:
Oops, yeah will revert later today

Reply:
@brandonSc fixed!